### DISCUSSION TITLE
Excellent library

### DISCUSSION THREAD

**Discussion by ihahachi** [clickbar/laravel-magellan#59] [Q&A] [answered]
Hello,

First of all, I want to express my appreciation for this excellent library.
I want to create a platform to uploading a GeoJSON or KML file and  save it to PostGIS
Is there documentation about the Generators & Parsers?

**Comment by ahawlitschek** [comment, marked as answer]
Thanks a lot for the kind words!
Currently we don't have a generator or parser for KML. 
We only support WKT, WKB and GeoJSON geometries. 

Since KML as well as GeoJSON support more than plain geometry data I assume you also want to use these additional data. 
Am I right?
